scripts/setup/setup-certbot

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage() {
+  echo "Usage: $0 <domain name> <email>" >&2
+  exit 1
+}
+
+if [ "$EUID" -ne 0 ]; then
+    echo "Error: This script must be run as root" >&2
+    exit 1
+fi
+
+if [ "$#" -ne 2 ]; then
+    usage
+fi
+
+DOMAIN="$1"
+EMAIL="$2"
+
+# For reference https://certbot.eff.org/all-instructions/#ubuntu-14-04-trusty-nginx
+apt-get update
+apt-get install -y software-properties-common
+add-apt-repository -y ppa:certbot/certbot
+apt-get update
+apt-get install -y python-certbot-nginx
+
+certbot --nginx -d "$DOMAIN" -m "$EMAIL" --agree-tos --non-interactive
+
+# Link the generated cert to the path read by Zulip
+ln -nsf /etc/letsencrypt/live/"$DOMAIN"/privkey.pem /etc/ssl/private/zulip.key
+ln -nsf /etc/letsencrypt/live/"$DOMAIN"/fullchain.pem /etc/ssl/certs/zulip.combined-chain.crt
+
+
```

Comment:
probably we should echo a success message for this step.

Reply:
Addressed